internal/output/output.go

```diff
@@ -45,6 +46,72 @@ type Project struct {
 	fullSummary   *Summary
 }
 
+// ToSchemaProject generates a schema.Project from a Project. The created schema.Project is not suitable to be
+// used outside simple schema.Project to schema.Project comparisons. It contains missing information
+// that cannot be inferred from a Project.
+func (p Project) ToSchemaProject() *schema.Project {
```

Comment:
Do we still need these functions?

Reply:
Yup, they're used in the `output.CompareTo` function

Reply:
Ah, because our diff functionality uses schema.Resource?

Reply:
yeah